scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -0,0 +1,31 @@
+namespace <%= namespace %>.SDK {
+    using System.Collections.Generic;
+    using MiniJSON;
+
+    public class ShopifyError {
+        public enum ErrorType {
+            HTTP, GraphQL
+        }
+
+        public readonly ErrorType error;
+        public readonly string description;
+
+        public ShopifyError(ErrorType error, string description) {
+            this.error = error;
+            this.description = description;
+        }
+
+        public static explicit operator ShopifyError(TopLevelResponse response) {
+            if (response.HTTPError != null) {
+                return new ShopifyError(ShopifyError.ErrorType.HTTP, response.HTTPError);
+            } 
+            
+            if (response.errors != null && response.errors.Count > 0) {
+                string graphQLErrorsString = Json.Serialize(response.errors);
```

Comment:
In this PR could you refactor `TopLevelResponse` to simply serialize the `errors` as a string. Basically change.
`private List<string> ParseErrors(Dictionary<string,object> dataJSON) {`
to
`private string ParseErrors(Dictionary<string,object> dataJSON) {`